lisa/core/node.py

```diff
@@ -98,6 +100,19 @@ def setConnectionInfo(
         self.internalAddress = address
         self.internalPort = port
 
+    def getScriptPath(self) -> pathlib.Path:
+        assert self.workingPath
+        script_path = self.workingPath.joinpath(constants.PATH_SCRIPT)
+        return script_path
+
+    def getScript(self, script_spec: CustomScriptSpec) -> CustomScript:
+        # register on node to prevent copy files again in another test suite
```

Comment:
Should we need to do this? It should be really easy (read: fast, and accurate) to check for the script's presence before uploading it.

Reply:
The script may have the same name, and be copied to same place incorrectly, my current idea is to overwrite it, and try best to make it right. Another things is any remote operation is time costing, so caching it to save time.

Reply:
So we want to upload if and if only if it's different? We could use `rsync` to do that quickly.

Reply:
Feel free to integrate rsync, and make sure it works on Windows. The installation of rsync is better in our code, instead of an external prequisites.